src/ConEmu/SetPgSizePos.cpp

```diff
@@ -89,7 +89,7 @@ LRESULT CSetPgSizePos::OnInitDialog(HWND hDlg, bool abInitial)
 
 	checkDlgButton(hDlg, cbLongOutput, gpSet->AutoBufferHeight);
 	TODO("Надо бы увеличить, но нужно сервер допиливать");
-	SendDlgItemMessage(hDlg, tLongOutputHeight, EM_SETLIMITTEXT, 5, 0);
+	SendDlgItemMessage(hDlg, tLongOutputHeight, EM_SETLIMITTEXT, 6, 0);
```

Comment:
Than you should rollback this change

Reply:
Hi, see my reply at https://github.com/Maximus5/ConEmu/pull/1604#issuecomment-397253345

This can't be rolled back without breaking the UI when the user enters 222222 or 111111. Users do that when they want to set the output height to maximum value (which is 32766 in this case).